### ISSUE TITLE
TSC Meeting (June 1, 2021), v3.5.0

### ISSUE THREAD

**Issue by amanda11** [StackStorm/discussions#79]
June @StackStorm/tsc `1 hour` meeting will take place on Tuesday, `1st June 2021, 09:30 AM US Pacific`.
See #33 for more info about how to join.

## Meeting Agenda (TBC)

# Agenda

## StackStorm v3.5.0 Status (@amanda11)

  Details:
    - Roadmap: https://github.com/orgs/StackStorm/projects/20
    - Release Manager @amanda11, Release Assistant: @winem
    - Target: June
  - Add Python 3.8 support (@winem, @carlos, @kami)
  - Ubuntu 20 Migration https://github.com/StackStorm/discussions/issues/68 (@winem, @carlos)
  - Remove support for Ubuntu 16: https://github.com/StackStorm/discussions/issues/69
  - Core Performance Improvements (done) - @Kami
  - Redis as a default st2 coordination backend https://github.com/StackStorm/discussions/issues/70 (@m4dcoder)
    - Ansible Deployment (done) - @amanda11
    - Puppet Deployment - @nmaludy
    - Docker & K8s (done) - @armab
  - PIP & Virtualenv Maintenance upgrade (done) - @amanda11
  - RBAC/LDAP support in Ansible Deployment - (@winem)
  - Node upgrade?

### deb/rpm Package Hosting solutions (@nmaludy, @blag?)
  - Sub-committee WIP: https://github.com/StackStorm/discussions/issues/74

### Migrating stackstorm.com from Wordpress to static github pages (@armab, @nmaludy)
  - Sub-committee WIP: https://github.com/StackStorm/discussions/issues/76

### Migrating forum.stackstorm.com to Github Discussions (no owner)
  - Sub-committee WIP: https://github.com/StackStorm/discussions/issues/75
  - GH Discussions: https://github.com/StackStorm/st2/discussions

### Partner Program Progress (Mick)
  - Sub-committee WIP: https://github.com/StackStorm/discussions/issues/73
  - Plan: finalize the details, bootstrap and announce the program to the public

**Comment by mickmcgrath13** [comment]
For the Partner Program, updates are here:
https://github.com/StackStorm/discussions/issues/73#issuecomment-852177822

tldr, next steps are:
- Submit formal proposal for TSC voting (this week)
- Make it live! (i.e. Page swapping https://stackstorm.com/stackstorm-partners-program-description/?preview=true from https://stackstorm.com/partners-program/) (next week ish)

**Comment by amanda11** [comment]
# Meeting Notes #

* 3.5 Release
  - Target code freeze date Monday 14th June
  - Reviewed 3.5 project plan
     - U20 updates done. Query on UT for Mongo 4.4 - to be chased offline. Ankur volunteered to look at puppet changes. Marcel started looking at ansible changes. Puppet/Ansible changes don't block release date.
     - Review mongo 4.4/4.2 support across all O/Ses for 3.6 project plan, as well as redis and rabbitMQ versions
     - U16 - main items needed are cicd (@amanda11) to handle, documentation and scripts. overnight U16 unstable runs stopped.
     - redis - only documentation is blocking release - @m4dcoder to update
     - TLS v1.3 - @punkrokk to look at making it the default
     - st2web issue on blank screen to move to 3.6 release
     - nodejs - will not block release if don't get upgrade to node 14 working. Blocking at moment with st2chatops build with node 14 - see PR https://github.com/StackStorm/st2chatops/pull/161. Help required with node/chatops experience.
     - st2packer to move from xenial to bionic - @armab volunteered to look at
     - upgrade notes for migration tool - @cognifloyd to review @Kami's doc changes on https://github.com/StackStorm/st2docs/pull/1071


* Sub-commitee updates
 - PackageCloud. @blag had spoken about their offering. @nmaludy had had follow-up with CloudSmith who are willing to offer free plan, but details/requirements to be clarified
 - WordPress move - no update
 - Forum STackStorm -> GitHub discussions - GitHub discussions still Beta, so waiting a few months
 - Partner Program - @raviorch gave overview of the partners program that the sub-committee had worked on. To be raised as a GitHub discussion so that the TSC can vote asyncrhonously
